4_extract.R

```diff
@@ -0,0 +1,16 @@
+source('4_extract/src/extract_model_data.R')
+
+p4 <- list(
+  tar_target(
+    p4_extract_uncal_models_feather,
+    extra_glm_output(p2_nldas_glm_uncal_tibble),
+    format = 'file'
```

Comment:
SC: As is, I can't tell from looking at this target recipe where the output files are going to be saved.  In these situations where the output filepath needs to be customized within the function, my preference is to include a [output file template](https://github.com/USGS-R/lake-temperature-process-models/blob/main/3_extract.R#L77), e.g., '4_extract/out/%s/%.feather' which could then be customized with the `run_type` and site_id within the function. At the least, I'd pass in `dir_out` as an argument so that the user would know where to look for the output files

Reply:
Thanks for the suggestion. I implemented this